src/main/java/io/seqera/tower/cli/commands/runs/DumpCmd.java

```diff
@@ -112,7 +112,10 @@ protected Response exec() throws ApiException, IOException {
             tar.add("workflow-launch.json", collectWorkflowLaunch(wspId));
```

Comment:
Let's handle the null value here to keep consistency and to avoid creating an empty file.

```suggestion

            var launchData = collectWorkflowLaunch(wspId);
            if (launchData != null) {
                tar.add("workflow-launch.json", launchData);
            } else {
                progress.println(ansi("\t- No data collected, skipping")); // nextflow-run workflows doesn't have launches
            }

```

Reply:
[1f5a5d6](https://github.com/seqeralabs/tower-cli/pull/520/commits/1f5a5d69d39022f2c08d1cfd0edc01dce0723bb3)